pkg/test/runner/runner.go

```diff
@@ -52,6 +52,9 @@ const (
 	expectedResultsFile string = "results.yaml"
 	expectedDiffFile    string = "diff.patch"
 	expectedConfigFile  string = "config.yaml"
+	setupScript         string = "setup.sh"
```

Comment:
Where do you document on how to use these scripts?

Reply:
Will add a readme for e2e test runner

Reply:
done